src/main/groovy/org/labkey/gradle/task/RunTestSuite.groovy

```diff
@@ -42,8 +44,6 @@ class RunTestSuite extends RunUiTest
             dependsOn(project.project(':tools:Rpackages:install'))
         if (!project.getPlugins().hasPlugin(TeamCity.class) && project.tasks.findByName('packageChromeExtensions') != null)
             dependsOn(project.tasks.packageChromeExtensions)
-        if (project.findProject(":tools:Rpackages:install") != null)
-            dependsOn(project.project(':tools:Rpackages:install'))
```

Comment:
@labkey-tchad  When did this dependency on Rpackages go away?

Reply:
It's declared few lines earlier. I was removing the duplicate.